src/app.ts

```diff
@@ -26,60 +25,64 @@ const jsonKeyring = readFileSync(KEYRING_PATH).toString();
 const KEYRING = GearKeyring.fromJson(jsonKeyring, KEYRING_PASSPHRASE);
 
 const createVoucher = async (accountUser: HexString) => {
+  const provider = new WsProvider(NODE_ADDRESS);
```

Comment:
Since there's no problem with `create` method itself, I would stick to initilization with a `providerAddress` parameter